application/fin/netsuite_test.go

```diff
@@ -51,10 +52,8 @@ func TestNetSuite_Export(t *testing.T) {
 		TransactionBlocks:  TransactionBlocks{"": Transactions{t1}, "bar": Transactions{t2}},
 	}
 
-	summaryRow := fmt.Sprintf(`"1","000000","00001","","GL","JE","%d","%02d",0,"%s","00",0,0,0,2`+"\n",
-		n.Year, n.Period, n.JournalDescription)
-
 	transaction1Row := fmt.Sprintf(netSuiteTransactionRowTemplate,
+		fmt.Sprintf("%d%02d0%06d", n.Year, n.Period, 1),
```

Comment:
What do you think of this? I think it feels more like test code this way.
```suggestion
		"2009090000001",
```